docs/docs/dev-manuals/services/shipment.mdx

```diff
@@ -93,6 +141,8 @@ enum ShipmentStatus {
 }
 ```
 
+````
```

Comment:
I think this shouldn't be here...
This and line 162 cause the section to be formatted as some sort of code block.

Reply:
fixed with 5cb6c4b68ae5261f254cdc5392f3bf7c0a1560e7